src/logger/loggerApi.c

```diff
@@ -643,7 +643,7 @@ int api_getAnalogConfig(Serial *serial, const jsmntok_t * json){
 			startIndex = endIndex = modp_atoi(json->data);
 		}
 	}
-	if (startIndex >= 0 && startIndex <= CONFIG_ADC_CHANNELS){
+	if (startIndex <= CONFIG_ADC_CHANNELS){
 		sendAnalogConfig(serial, startIndex, endIndex);
```

Comment:
Its an unsigned variable where we are checking if its value is less than 0.  So more of cleanup.  CLANG was complaining about it.  Its not needed.